src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write code here
+const tbody = document.querySelector('tbody');
+const theadList = [...document.querySelector('thead tr').children];
+const rows = [...tbody.rows];
+
+document.body.addEventListener('click', e => {
+  const index = theadList.indexOf(e.target);
+
+  function salaryToNumber(sal) {
+    return +sal.slice(1).split(',').join('');
+  };
```

Comment:
It is better to move this function outside of eventListener, so that you can reuse it somewhere else if needed